src/profile-logic/process-profile.js

```diff
@@ -260,15 +265,21 @@ function _extractUnsymbolicatedFunction(
   } = extractionInfo;
 
   let resourceIndex = -1;
-  let addressRelativeToLib = -1;
+  let addressRelativeToLib: Address = -1;
+
+  // The frame address, as observed in the profiled process. This address was
```

Comment:
Thanks for the clarifications, this makes it much easier to understand what's going on here.